<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addproduct</class>
 <widget class="QDialog" name="addproduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameline">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="factoryline">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dayline">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Factory :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Category :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>License :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Price :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceline">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthline">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearline">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Adding new product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newproaddthisbutton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD THIS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantotyline">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="categorycombo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-----</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Drink</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Snack</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Food</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dairy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Others</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameline</tabstop>
  <tabstop>factoryline</tabstop>
  <tabstop>categorycombo</tabstop>
  <tabstop>dayline</tabstop>
  <tabstop>monthline</tabstop>
  <tabstop>yearline</tabstop>
  <tabstop>priceline</tabstop>
  <tabstop>quantotyline</tabstop>
  <tabstop>newproaddthisbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
